src/Testing/src/xunit/HelixConstants.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+
+// Do not change this namespace without changing the usage in ConditionalTheoryAttribute
```

Comment:
This is just a vanilla const, no? The namespace isn't hardcoded anywhere, is it?

Reply:
Oops, duped comments from another class I copied.